config/jobs/kubernetes/sig-testing/coverage.yaml

```diff
@@ -51,3 +56,66 @@ periodics:
         value: "n"
       securityContext:
         privileged: true
+# coverage for same tests that run in ci-kubernetes-e2e-gci-gce
+- interval: 1h
+  name: ci-kubernetes-coverage-e2e-gci-gce
+  decorate: true
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  - org: kubernetes
+    repo: release
+    base_ref: master
+    path_alias: k8s.io/release
+  decoration_config:
+    timeout: 10800000000000 # 3 hours
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-master
+      command:
+      - runner.sh
+      - bash
+      args:
+      - -e
+      - -c
+      - |
+        shopt -s globstar
```

Comment:
we're definitely going to regret this thing someday

Reply:
yeah I know, this is not a great pattern to be following and getting to N=3 means it's time for us to refactor, this is mostly curiosity-driven

Reply:
https://github.com/kubernetes/test-infra/pull/9392#discussion_r220306523

/this-is-fine